recipe/meta.yaml

```diff
@@ -1,17 +1,12 @@
-{% set version = "6.0.0rc5" %}
-
 package:
   name: gmt
-  version: {{ version }}
-
+  version: 6.2.0.dev0+a1a189c
 source:
-  url: https://github.com/GenericMappingTools/gmt/releases/download/{{ version }}/gmt-{{ version }}-src.tar.gz
-  sha256: 10154b216dfc290b95700b6620ef1c89b170074c14173fd585c30f980e270f38
-
+  git_url: https://github.com/GenericMappingTools/gmt.git
+  git_rev: a1a189cafe784c8dc9e3777e2f17a78c2885948d
```

Comment:
Ping @seisman / @leouieda , please review this and approve if you have time. This would make testing the PyGMT master branch a lot easier locally, and help with #100 too.